v2.0/revoke-roles.md

```diff
@@ -0,0 +1,84 @@
+---
+title: REVOKE &lt;roles&gt;
+summary: The REVOKE <roles> statement revokes privileges from users and/or roles.
+toc: false
+---
+
+<span class="version-tag">New in v2.0:</span> The `REVOKE <roles>` [statement](sql-statements.html) lets you revoke a [role](roles.html) or [user's](create-and-manage-users.html) membership to a role.
+
+{{site.data.alerts.callout_info}}You must have an <a href="enterprise-licensing.html">enterprise license</a> to remove a member from a role.{{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+## Synopsis
+
+<section>{% include sql/{{ page.version.version }}/diagrams/revoke_roles.html %}</section>
+
+## Required Privileges
+
+The user revoking role membership must be a role admin (i.e., members with the `ADMIN OPTION`) or a superuser (i.e., a member of the `admin` role).
+
+## Considerations
+
+- The `root` user can not be revoked from the `admin` role.
+
+## Parameters
+
+Parameter | Description
+----------|------------
+`role_name` | The name of the role from which you want to remove members. To revoke members from multiple roles, use a comma-separated list of role names.
+`user_name` | The name of the [user](create-and-manage-users.html) or [role](roles.html) from whom you want to revoke membership. To revoke multiple members, use a comma-separated list of user and/or role names.
+
+## Examples
+
+### Revoke role membership
+
+{% include copy-clipboard.html %}
```

Comment:
We should show the output of `SHOW GRANTS ON ROLE design;` before revoking the membership, otherwise it's hard to tell that the revoke did something.